VAMobile/src/components/FormWrapper/FormFields/Picker/VAModalPicker.test.tsx

```diff
@@ -84,21 +80,21 @@ context('VAModalPicker', () => {
   describe('when an option is selected', () => {
     it('should update selected to the value of that option and select done', async () => {
       await waitFor(() => {
-        selectionButtons[0].props.onPress()
+        fireEvent.press(component.getByTestId('Java'))
       })
 
       await waitFor(() => {
-        doneButton.props.onPress()
+        fireEvent.press(doneButton)
```

Comment:
replace with fireEvent.press(screen.getByRole('button, { name: 'Done' })) and remove the done button usage

Reply:
Updated.